css/types/color.json

```
@@ -319,7 +319,14 @@
                 "version_added": false
               },
               "safari": {
-                "version_added": false
+                "version_added": "15",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "CSS color-mix()",
+                    "value_to_set": "true"
```

IIRC, `value_to_set` is optional. Since there's no actual value to choose in the Safari UI, how about we omit it?

```suggestion
                    "name": "CSS color-mix()"
```

---

Hi @ddbeck, I was just following the pattern I saw here:

https://github.com/mdn/browser-compat-data/blob/db3cb3ec3984f24f277daa25a28eb15b2c45f936/css/types/color.json#L256

Do you want me to remove it from `color-contrast()` as well?

---

Fixed :)